### ISSUE TITLE
Job succesfully finishes in case if SLS can't be rendered

### ISSUE THREAD

**Issue by vitalyisaev2** [jenkinsci/saltstack-plugin#29]
Hi, it's me again :) 

I have noticed that plugin treats Jenkins job as successfully finished when target SLS file even couldn't be rendered...

```
Started by upstream project "DOCKER_IMAGE_centrifugo" build number 21
originally caused by:
 Started by upstream project "LOCAL_PROJECT_shilkin_centrifugo" build number 19
 originally caused by:
  Started by an SCM change
[EnvInject] - Loading node environment variables.
Building remotely on deployer (deploy) in workspace /var/lib/jenkins/workspace/DEPLOY_centrifugo
Sending JSON: [{"client":"local","tgt":"p-3-testmap.pio.tv.ro","expr_form":"glob","fun":"state.sls","arg":["mailpaas.centrifugo"]}]
Running jid: 20160510124113361338
Waiting for 1 minions
0 minions are done
Will check status every 10 seconds...
Response on state.sls mailpaas.centrifugo for p-3-testmap.pio.tv.ro:
[{
  "outputter": "highstate",
  "data": {"p-3-testmap.pio.tv.ro": ["Rendering SLS 'base:mailpaas.centrifugo' failed: mapping values are not allowed here; line 40\n\n---\n[...]\n    - ports: [{8001: 8001}]\n    - network_mode: host\n    - environment:\n      - SERVICE_NAME: centrifugo\n      - SERVICE_TAGS: testing\n    - command: --port=8001 --engine=tarantool --tt_aggr_node=testing.cocaine-agregate-node.service.consul --tt_aggr_port=10053 --tt_service_name={'subsystem': 'mail', 'binary': 'ttconnector', 'app': 'mailpaas-ttconnector', 'group': 'ttconnector'} --tt_acl=dummy_acl --tt_host=testing.tarantool-notification.service.consul --tt_port=33013 --tt_user=test --tt_password=123 --log_level=debug    <======================\n    - require:\n      - docker: centrifugo_image\n\u0000\n---"]}
}]
Finished: SUCCESS
```

**Comment by mchugh19** [comment]
Same for this one. Are you able to replicate this issue on 1.7.1? This was likely fixed in 0213ec9 which was released in 1.7.0.

**Comment by vitalyisaev2** [comment]
@mchugh19 this error cannot be reproduced too. Closing this one.
